packages/opentelemetry-metrics/src/types.ts

```diff
@@ -44,7 +37,7 @@ export interface MeterConfig {
 
 /** Default Meter configuration. */
 export const DEFAULT_CONFIG = {
-  logLevel: getEnv().OTEL_LOG_LEVEL,
+  diagLogLevel: getEnv().OTEL_LOG_LEVEL,
```

Comment:
should this be removed ?

Reply:
Nice catches, I obviously missed diagLogLevel removals -- will update (remove) all.